From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 1 Sep 2017 18:54:41 +0000
Subject: [PATCH] Only create yp library if enabled

Create the yellow pages lookup library symlink only when built with
yellow pages support.
---
 modules/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules/Makefile b/modules/Makefile
index 91dc20a..6908da0 100644
--- a/modules/Makefile
+++ b/modules/Makefile
@@ -77,7 +77,9 @@ install: all
 	install -c $(MODS) -m 755 $(INSTALLROOT)$(autofslibdir)
 	-rm -f $(INSTALLROOT)$(autofslibdir)/mount_smbfs.so
 	ln -fs lookup_file.so $(INSTALLROOT)$(autofslibdir)/lookup_files.so
+ifeq ($(YPCLNT), 1)
 	ln -fs lookup_yp.so $(INSTALLROOT)$(autofslibdir)/lookup_nis.so
+endif
 ifeq ($(LDAP), 1)
 	ln -fs lookup_ldap.so $(INSTALLROOT)$(autofslibdir)/lookup_ldaps.so
 endif
